CleanWeather/Scenes/CityList/CityListController.swift

```diff
@@ -71,6 +70,7 @@ extension CityListController: UITableViewDelegate {
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let id = citiesWeatherDataSource[indexPath.row].id
+        tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
ogólnie bardzo rzadko korzysta się z APPLOwego zaznaczania cellek itp, bo one brzydkie są
(i tu ten nasz konflikt, bo pisze Ci to zeby powiedziec ze tego nie uzywamy, i radzę ci abys we wszyskich swoich cellkach w xibie dal SelectionMode - None - ale jednooczenie nie chce marnowac czasu na to zeby teraz robic jakies wlasne podswietlenia :) - przez co mozliwe ze apka bedzie brzydka bo nie bedzie podswietlen)